src/lsm/scan_tree.zig

```diff
@@ -572,9 +572,12 @@ fn ScanTreeLevelType(comptime ScanTree: type, comptime Storage: type) type {
                 checksum: u128,
                 read: Grid.Read = undefined,
             },
-            iterating_values: struct {
+            iterating: struct {
```

Comment:
Now we've dropped `_values`, we could also rename `iterating_manifest`.

:thinking:  "iterating manifest" isn't accurate for what we are doing, maybe `loading_manifest`, `moving_manifest` , or just `loading`.